TACC/protx-dashboard
# Task/bump plotly

## Overview: ##

Bump plotly libraries on client and backend to fix a plotting bug.
Note, other libraries were updated on the client side as well.

## Related Jira tickets: ##

None

## Testing Steps: ##
1. Check that bug is gone
2. Check other areas of applications as so many libraries were updated

## UI Photos:

Previously:
<img width="550" alt="Screenshot 2023-01-17 at 4 44 13 PM" src="https://user-images.githubusercontent.com/8287580/213842136-0a21604f-0823-4283-adbd-3465068f05ff.png">

Update:
![Screenshot 2023-01-20 at 4 19 58 PM](https://user-images.githubusercontent.com/8287580/213842121-48491c6e-b955-4bb9-95c5-6f7e6bdc95a8.png)


## Notes: ##

TODO: 
- [ ] looks like the demographic chart is taking up too much vertical space
